### ISSUE TITLE
Sorry to bother you. Did you use mse as the loss function? My EWC is useless when I use MSE. Thanks a lot! 

### ISSUE THREAD

**Issue by Cat-Guijun** [zhang-xd18/cfnet#2]

**Comment by zhang-xd18** [comment]
As shown in the paper, the loss function consists of two parts: the MSE loss of channel matrices and the regularization term that defeats knowledge forgetting.
I hope this can answer your question.

**Comment by Cat-Guijun** [comment]
Thanks for your reply! My reproduced code is hard to be useful. 
Can you kindly offer me some reference code link? 
If I just replace the softmax function by mse in github codes, it's useless.
Thanks a lot!

**Comment by zhang-xd18** [comment]
I think maybe I will recommend you refer to this repository: https://github.com/joansj/hat.

**Comment by Cat-Guijun** [comment]
Thank you very much!
